Library/ViewModels/SettingsViewModel.swift

```diff
@@ -576,6 +611,34 @@ SettingsViewModelOutputs {
   public var outputs: SettingsViewModelOutputs { return self }
 }
 
+private func dateFormatter(for dateString: String?, state: ExportDataEnvelope.State) -> String {
+  let dateFormatter = DateFormatter()
+  dateFormatter.dateFormat = "yyyy-MM-dd"
+  let newDateFormatter = DateFormatter()
+  newDateFormatter.dateFormat = "MMM d"
+
+  let timeFormatter = DateFormatter()
+  timeFormatter.dateFormat = "HH-mm-sZ"
+  let newTimeFormatter = DateFormatter()
+  newTimeFormatter.timeZone = TimeZone(secondsFromGMT: TimeZone.current.secondsFromGMT())
+  newTimeFormatter.dateFormat = "h:mm a"
```

Comment:
It looks like we might already have a helper function that does a lot of the heavy lifting with date formatting, and caches the formatters whenever possible (this is good for performance). `Format.date(seconds: secondsFromUTC, template: "h: mm a", timeZone: TimeZone.current)` could work, but first you'd have to convert the `dateString` into a `Date` object. It looks like those helper functions (in `Format.swift`) don't currently have a helper that goes from `String -> Date`, but we could always add one. I think converting the `dateString` to a `Date` first could be a better approach here because then you don't have to do all the string parsing manually, which could be error-prone. Happy to pair on this with you if you like!